components/Graphs/StarsChart.tsx

```diff
@@ -0,0 +1,44 @@
+import { FaStar } from "react-icons/fa6";
+import { ResponsiveContainer, BarChart, XAxis, YAxis, Bar, Tooltip, type TooltipProps } from "recharts";
+import { ValueType, NameType } from "recharts/types/component/DefaultTooltipContent";
+import Card from "components/atoms/Card/card";
+
+import { getStarsHistogramToDays } from "lib/utils/repo-page-utils";
+import { type StatsType } from "lib/hooks/api/useFetchMetricStats";
+
+type StarsChartProps = {
+  stats: StatsType[] | undefined;
+  syncId: string;
+  range: number;
+};
+
+export default function StarsChart({ stats, syncId, range = 30 }: StarsChartProps) {
+  const chartData = getStarsHistogramToDays({ stats, range });
+  return (
+    <Card className="flex flex-col gap-8 w-full lg:min-w-[64rem] h-full items-center pt-8">
```

Comment:
Add a title that is composed by the metric's name + the time range currently selected like:

```
<h3 class="font-semibold text-lg text-slate-800 w-full">
   Stars <span class="pl-1 pl-2 text-slate-500 text-sm font-medium"> 90 days</span> 
</h3>
```